### PR TITLE
Used TestRunner

### PR DESCRIPTION
**Pull request by Nikodych**

### CODE REVIEW

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java [hunk 1]
```diff
@@ -1,69 +1,45 @@
...
-    @BeforeClass
-    final public void setUp() {
-        driver = new WebDriverFactory().getDriver(Drivers.CHROME);
-        driver.manage().window().maximize();
-        driver.manage().timeouts().implicitlyWait(TIMEOUT, TimeUnit.SECONDS);
-        driver.get(LOG_IN_PAGE);
+    //  This test checks if Administrator user can see the following tabs: ‘Administration’ and ‘User Info’(default),
+    //  test checks if Administrator user can switch between tabs.
+    @Test
+    final public void testSwitchingBetweenTabs() {
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java:24]: STICK

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java [hunk 2]
```diff
@@ -1,69 +1,45 @@
-package academy.softserve.edu.tests;
+package academy.softserve.edu.tests.administrator;
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java:1]: ADD MESSAGES TO ALL ASSERTS!

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java (hunk 1 above)
Comment (Nikodych) [reply src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java:24]: fixed

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java (hunk 2 above)
Comment (Nikodych) [reply src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java:1]: done

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import static academy.softserve.edu.pageobjects.UserInfoPage.ADMINISTRATION_LINK;
+import static academy.softserve.edu.pageobjects.UserInfoPage.USER_INFO_LINK;
+import static academy.softserve.edu.pageobjects.UserInfoPage.USER_INFO_PAGE_URL;
+
+@Listeners({TestListener.class})
+public class AdminAfterLogInPageTest extends TestRunner {
+
+    private static final String ADMIN_USERNAME = "iva";
+    private static final String ADMIN_PASSWORD = "qwerty";
+
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java:19]: move out to test runner class

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    //  This test checks if Administrator user can see the following tabs: ‘Administration’ and ‘User Info’(default),
+    //  test checks if Administrator user can switch between tabs.
+    @Test
+    final public void testSwitchingBetweenTabs() {
+
+        final LogInPage logInPage = new LogInPage(driver);
+        logInPage.doLogIn(ADMIN_USERNAME, ADMIN_PASSWORD);
+        Assert.assertEquals(driver
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java]: avoid using URLs for testing

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                .getCurrentUrl(), USER_INFO_PAGE_URL, "URL is different from " + USER_INFO_PAGE_URL);
+
+        UserInfoPage userInfoPage = new UserInfoPage(driver);
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java]: STICK

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java (hunk 3 above)
Comment (Nikodych) [reply src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java:19]: done

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java (hunk 4 above)
Comment (Nikodych) [reply src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java]: done

File: src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java (hunk 5 above)
Comment (Nikodych) [reply src/test/java/academy/softserve/edu/tests/administrator/AdminAfterLogInPageTest.java]: done

**Pull request merged**